bde_env.drush.inc

```diff
@@ -82,7 +82,18 @@ function drush_bde_env_gen($filename = '') {
     ),
   );
 
-  $output = 'export BEHAT_PARAMS=' . escapeshellarg(json_encode($parameters, JSON_UNESCAPED_SLASHES));
+  switch (getenv('SHELL')) {
+    case '/usr/bin/fish':
+      $prefix = 'set -U -x BEHAT_PARAMS ';
+      break;
+
+    default:
+      $prefix = 'export BEHAT_PARAMS=';
+      break;
+
+  }
+
```

Comment:
Oops my context line seems to have gone missing. I meant the one above the closing curly brace.